core/oslc4j-core/pom.xml

```diff
@@ -70,6 +65,15 @@
       <artifactId>jakarta.xml.bind-api</artifactId>
     </dependency>
 
+    <dependency>
+        <groupId>org.apache.wink</groupId>
+        <artifactId>wink-json4j</artifactId>
+    </dependency>
```

Comment:
we should not bring Wink back into non-legacy Lyo modules

Reply:
This Wink has always been there.